docs/eventing/getting-started.md

```diff
@@ -0,0 +1,528 @@
+the `type` and `source` attributes. The `type` and `source` attributes are used by each Trigger to match events 
+to consumers, where a trigger's `filter` specifies the event `type` attribute, and a trigger's `subscriber` defines the event `source` attribute.
+
+
+1. To create the first `Trigger`, run the following command:
+
+    ```sh
+    kubectl --namespace event-example apply --filename - << END
+    apiVersion: eventing.knative.dev/v1alpha1
+    kind: Trigger
+    metadata:
+      name: hello-display
+    spec:
+      filter:
+        sourceAndType:
+          type: greeting
+      subscriber:
+        ref:
+         apiVersion: v1
+         kind: Service
+         name: hello-display
+    END
+    ```
+
+   Result: You created a trigger that sends all events of type `greeting` to your event consumer named `hello-display`.
+
+2. To add the second `Trigger`, run the following command:
+
+    ```sh
+    kubectl --namespace event-example apply --filename - << END
+    apiVersion: eventing.knative.dev/v1alpha1
+    kind: Trigger
+    metadata:
+      name: goodbye-display
+    spec:
+      filter:
+        sourceAndType:
+          source: sendoff
+      subscriber:
+        ref:
+         apiVersion: v1
+         kind: Service
+         name: goodbye-display
+    END
+    ```
+
+   Result: You created a trigger that sends all events of source `sendoff` to your event consumer named `goodbye-display`.
+
+3. Verify that the triggers are working correctly by running the following command:
+
+    ```sh
+    kubectl --namespace event-example get triggers
+    ```
+
+    Result: Both the `hello-display` and `goodbye-display` triggers that you created should be listed, for example:
+
+
+    ```sh
+    NAME                   READY   REASON   BROKER    SUBSCRIBER_URI                                                                 AGE
+    goodbye-display        True             default   http://goodbye-display.event-example.svc.cluster.local/                        9s
+    hello-display          True             default   http://hello-display.event-example.svc.cluster.local/                          16s
+    ```
+
+    Both triggers should be ready and pointing to the correct **Broker** (your triggers should be pointing at the default broker)  and **SUBSCRIBER_URI** (your triggers should be pointing to triggerName.namespaceName.svc.cluster.local). If this is not the case, see the [Debugging Guide](./debugging/README.md) to help troubleshoot the issue.
+
+You have now created all of the resources needed to receive and manage events. You created the `Broker`, which manages the events sent to event consumers with the help of triggers.  In the next section, you will make the event producer that will be used to create your events.
+
+
+
+### Creating event producers
+
+In this section you will create an event producer that you can use to interact with the Knative Eventing subcomponents you created earlier.
```

Comment:
It looks like you have some paragraphs here where you've started a new line at the end, but you need one empty line between paragraphs for proper markdown style.